### PR TITLE
Update AR templates to use Update 11 references

### PR DESCRIPTION
**Pull request by dotMorten**

### CODE REVIEW

File: ProjectTemplates/ArcGISRuntime.AR/Installer/source.extension.vsixmanifest [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
     <Metadata>
-        <Identity Id="Esri.ArcGISRuntime.ARTemplates" Version="1.2.0" Language="en-US" Publisher="Esri" />
+        <Identity Id="Esri.ArcGISRuntime.ARTemplates" Version="1.3.0" Language="en-US" Publisher="Esri" />
         <DisplayName>ArcGIS Runtime SDK for .NET - AR Templates</DisplayName>
         <Description xml:space="preserve">Project templates for building Augmented Reality apps for Android and iOS with Xamarin and ArcGIS Runtime SDK for .NET.</Description>
         <MoreInfo>https://links.esri.com/runtime-dotnet-sdk</MoreInfo>
         <License>EULA.rtf</License>
+        <ReleaseNotes>Updated to ArcGIS Runtime v100.11.0</ReleaseNotes>
```
Comment (mbranscomb) [inline ProjectTemplates/ArcGISRuntime.AR/Installer/source.extension.vsixmanifest]: Is this permitted in the spec?
> Specifies a file (.txt, .rtf) that contains release notes. Also takes the URL of a website that displays the release notes.

File: ProjectTemplates/ArcGISRuntime.AR/Template.Forms/Android/Android.csproj [hunk 2]
```diff
@@ -53,9 +53,8 @@
     <Reference Include="System.Xml" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.3.0.991211" />
-    <PackageReference Include="Xamarin.Android.Support.Core.Utils" Version="28.0.0.3" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.5.0" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.1874" />
```
Comment (mbranscomb) [inline ProjectTemplates/ArcGISRuntime.AR/Template.Forms/Android/Android.csproj:56]: Is it worth going straight to latest stable?

File: ProjectTemplates/ArcGISRuntime.AR/Template.Forms/App/AR.csproj [hunk 3]
```diff
@@ -6,9 +6,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Esri.ArcGISRuntime.ARToolkit.Forms" Version="100.7.1" />
-    <PackageReference Include="Xamarin.Forms" Version="4.3.0.991211" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.5.0" />
+    <PackageReference Include="Esri.ArcGISRuntime.ARToolkit.Forms" Version="100.11.0" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.1874" />
```
Comment (mbranscomb) [inline ProjectTemplates/ArcGISRuntime.AR/Template.Forms/App/AR.csproj:10]: Is it worth going straight to latest stable?

File: ProjectTemplates/ArcGISRuntime.AR/Template.Forms/iOS/iOS.csproj [hunk 4]
```diff
@@ -119,8 +119,8 @@
     <Reference Include="System.Numerics.Vectors" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Xamarin.Forms" Version="4.3.0.991211" />
-    <PackageReference Include="Xamarin.Essentials" Version="1.5.0" />
+    <PackageReference Include="Xamarin.Forms" Version="5.0.0.1874" />
```
Comment (mbranscomb) [inline ProjectTemplates/ArcGISRuntime.AR/Template.Forms/iOS/iOS.csproj:122]: Is it worth going straight to latest stable?

**Review by mbranscomb — CHANGES_REQUESTED** [review]
@dotMorten 
I'm not sure about including text in the vsixmanifest ReleaseNotes?

File: ProjectTemplates/ArcGISRuntime.AR/Template.Forms/Android/Android.csproj (hunk 2 above)
Comment (dotMorten) [reply ProjectTemplates/ArcGISRuntime.AR/Template.Forms/Android/Android.csproj:56]: It matches our requirements. So no. Users are free to pick a later version.

File: ProjectTemplates/ArcGISRuntime.AR/Installer/source.extension.vsixmanifest (hunk 1 above)
Comment (dotMorten) [reply ProjectTemplates/ArcGISRuntime.AR/Installer/source.extension.vsixmanifest]: Odd that it let me enter a string. I'll remove

**Review by mbranscomb — APPROVED** [review]

**Pull request merged**